modules/db_instance/variables.tf

```diff
@@ -249,6 +249,12 @@ variable "maintenance_window" {
   default     = null
 }
 
+variable "blue_green_update_enabled" {
```

Comment:
this will need to be updated for the change above https://github.com/terraform-aws-modules/terraform-aws-rds/pull/468/files#r1088005943